README.md

```diff
@@ -86,6 +86,23 @@ Finally, in case you are using banners and want to have further control on the a
 </div>
 ```
 
+### Overriding default opaqueUserId behavior
+
+If you want to pass your own `opaqueUserId` to the library, you can do so by overiding the `getUserID()` function which is
```

Comment:
Do we want to call `opaqueUserId` that way? Why not just `User ID`?
i think the idea of this library is to abstract away the complexities of the events API. Otherwise we would have to explain or link to docs the definition of `opaqueUserId`.

Reply:
Sure.